pyiceberg/catalog/glue.py

```diff
@@ -784,4 +784,4 @@ def drop_view(self, identifier: Union[str, Identifier]) -> None:
 
     @staticmethod
     def __is_iceberg_table(table: TableTypeDef) -> bool:
-        return table["Parameters"] is not None and table["Parameters"][TABLE_TYPE].lower() == ICEBERG
+        return table.get("Parameters", {}).get("table_type", "").lower() == "iceberg"
```

Comment:
One small request,
```suggestion
        return table.get("Parameters", {}).get("table_type", "").lower() == ICEBERG
```
we can use the ICEBERG constant defined in `catalog/__init__.py` here.